### DISCUSSION TITLE
About checking supplies during EDDI ship swapped event

### DISCUSSION THREAD

**Discussion by YenShihHung** [EDCD/EDDI#2674] [EDDI VoiceAttack Q&A] [answered]
I want to check the number of npc pilots and limpets when exchanging ships, but the EDDI ship loadout event occurs after the exchange event, so there is no way to check the data

Or can I only add a status flag and wait for the loadout event to perform this check?

**Comment by Tkael** [comment, marked as answer]
Neither of those items is affected by the `Ship loadout` event though you can certainly save a variable in VoiceAttack and save information for use at a later time via the `setstate` plugin function (https://github.com/EDCD/EDDI/wiki/VoiceAttack-Integration#setstate).

The `speech` plugin command (https://github.com/EDCD/EDDI/wiki/VoiceAttack-Integration#speech) can invoke the `Check limpets` Speech Responder script. You can look at EDDI.vap for examples showing how to use this plugin context.

At present EDDI does not track crew (since there is no way for us to validate that the crew information we've collected is complete).
